provision/modules/platform/platforms.tf

```diff
@@ -283,75 +283,83 @@ locals {
       "ssh_port" : 22
     },
     "rocky_8.5" : {
-      "ami_name" : "Rocky-8-ec2-8.5-202*.x86_64",
+      "ami_name" : "Rocky-8-EC2-8.5-*.x86_64",
       "owner" : "792107900819",
       "interface" : "eth0"
       "connection" : "ssh",
-      "ssh_user" : "ec2-user",
+      "ssh_user" : "rocky",
       "ssh_port" : 22
     },
     "rocky_8.6" : {
-      "ami_name" : "Rocky-8-ec2-8.6-202*.x86_64",
+      "ami_name" : "Rocky-8-ec2-8.6-*.x86_64",
       "owner" : "792107900819",
       "interface" : "eth0"
       "connection" : "ssh",
-      "ssh_user" : "ec2-user",
+      "ssh_user" : "rocky",
       "ssh_port" : 22
     },
     "rocky_8.7" : {
-      "ami_name" : "Rocky-8-EC2-8.7-202*.x86_64",
+      "ami_name" : "Rocky-8-EC2-Base-8.7-*.x86_64",
       "owner" : "792107900819",
       "interface" : "eth0"
       "connection" : "ssh",
-      "ssh_user" : "ec2-user",
+      "ssh_user" : "rocky",
       "ssh_port" : 22
     },
     "rocky_8.8" : {
-      "ami_name" : "Rocky-8-EC2-Base-8.8-202*.x86_64",
+      "ami_name" : "Rocky-8-EC2-Base-8.8-*.x86_64",
       "owner" : "792107900819",
       "interface" : "eth0"
       "connection" : "ssh",
-      "ssh_user" : "ec2-user",
+      "ssh_user" : "rocky",
       "ssh_port" : 22
     },
     "rocky_8" : {
-      "ami_name" : "Rocky-8-EC2-Base-8.8-202*.x86_64",
+      "ami_name" : "Rocky-8-EC2-Base-8.8-*.x86_64",
       "owner" : "792107900819",
       "interface" : "eth0"
       "connection" : "ssh",
-      "ssh_user" : "ec2-user",
+      "ssh_user" : "rocky",
       "ssh_port" : 22
     },
     "rocky_9.0" : {
-      "ami_name" : "Rocky-9-EC2-9.0-202*.x86_64",
+      "ami_name" : "Rocky-9-EC2-9.0-*.x86_64",
       "owner" : "792107900819",
       "interface" : "eth0"
       "connection" : "ssh",
-      "ssh_user" : "ec2-user",
+      "ssh_user" : "rocky",
       "ssh_port" : 22
     },
     "rocky_9.1" : {
-      "ami_name" : "Rocky-9-EC2-Base-9.1-202*.x86_64",
+      "ami_name" : "Rocky-9-EC2-Base-9.1-*.x86_64",
       "owner" : "792107900819",
       "interface" : "eth0"
       "connection" : "ssh",
-      "ssh_user" : "ec2-user",
+      "ssh_user" : "rocky",
       "ssh_port" : 22
     },
     "rocky_9.2" : {
-      "ami_name" : "Rocky-9-EC2-Base-9.2-202*.x86_64",
+      "ami_name" : "Rocky-9-EC2-Base-9.2-*.x86_64",
       "owner" : "792107900819",
       "interface" : "eth0"
       "connection" : "ssh",
-      "ssh_user" : "ec2-user",
+      "ssh_user" : "rocky",
+      "ssh_port" : 22
+    },
+    "rocky_9.3" : {
+      "ami_name" : "Rocky-9-EC2-Base-9.3-*.x86_64",
+      "owner" : "792107900819",
+      "interface" : "eth0"
+      "connection" : "ssh",
+      "ssh_user" : "rocky",
       "ssh_port" : 22
     },
     "rocky_9" : {
```

Comment:
I think `rocky_9` should be the latest rocky image, no? Right now it's set to 9.2 when there is a rocky 9.3 available

Reply:
good catch.  Let me actually see if I can make it even less picky.